apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/PoolRewrite/SessionPoolOptions.cs

```diff
@@ -0,0 +1,233 @@
+﻿// Copyright 2017 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Api.Gax;
+using Google.Api.Gax.Grpc;
+using System;
+
+namespace Google.Cloud.Spanner.V1.PoolRewrite
+{
+    /// <summary>
+    /// Options for session pools.
+    /// </summary>
+    public sealed class SessionPoolOptions
+    {
+        // Note: if any of these defaults are changed, update the documentation accordingly.
+        private int _maximumActiveSessions = 100;
+        private int _minimumPooledSessions = 10;
+        private TimeSpan _idleSessionRefreshDelay = TimeSpan.FromMinutes(15);
+        private TimeSpan _poolEvictionDelay = TimeSpan.FromDays(7);
+        private ResourcesExhaustedBehavior _waitOnResourcesExhausted = ResourcesExhaustedBehavior.Block;
+        private double _writeSessionsFraction;
```

Comment:
No default here? Should be .2 according to property docstring.

Reply:
I meant the XML comments that create the documentation. Will clarify here.

Reply:
I meant that `_writeSessionsFraction` has no default value set here where in the docstring for the `WriteSessionsFraction` says it should be `0.2`

Reply:
And OK for adding the XML document reference which was in another comment of mine.